src/data/scoreboard_config.py

```diff
@@ -164,13 +164,13 @@ def read_json(self, filename):
 
         j = {}
         path = get_file("config/{}".format(filename))
-        if os.path.isfile(path):
-            try:
-                j = json.load(open(path))
-                msg = "json loaded OK"
-            except json.decoder.JSONDecodeError as e:
-                msg = "Unable to load json: {0}".format(e)
-                j = {}
+        try:
```

Comment:
This `try..except` catches missing files, otherwise `msg` never gets set and leaves a cryptic error message